applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_static_solver.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, main_model_part, custom_settings):
             "clear_storage": false,
             "component_wise": false,
             "move_mesh_flag": true,
-            "convergence_criterion": "Residual_criteria",
+            "convergence_criterion": "Residual_criterion",
```

Comment:
I know this is not the point of this PR, but the capitalization of the different criteria are pretty strange (Uppercase "R" plus underscores?) I would be inclined to adopt the same conventions as for the json variable names at some point.